app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import { browserHistory } from 'react-router'
+import { isBrowser } from '../utils'
+
+const initialState = {
```

Comment:
Ah, that's interesting. Those are good practices, but they don't take into account async, which is why I do it the Tom way. Based on admittedly minimal experience, I think that if the view doesn't have an initial state (beyond null) and the redux is a bit behind render() because of an async call to the db, it's going to throw errors where properties in the render are 'undefined'. So, there's a potential 2ms display of an initial 'fake' state, but that's imperceptible to the eye and prevents the error. The way around this, I think, would be actions that account for the async, e.g., REQUEST_TREATMENT, then LOAD_TREATMENT, providing either an intermediate state in REQUEST... or some 'loading' prop that can be used to prevent the render() from accessing undefined state props until loading is set to false. Again, I have puddle depth of knowledge here, but the above explains my operating assumptions, whether they're actually right or mistaken!